Test1/task3New/main.c

```diff
@@ -0,0 +1,91 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+
+#define size 1000
```

Comment:
```suggestion
#define SIZE 1000
```
Макросы пишутся КАПСОМ_С_ПОДЧЁРКИВАНИЕМ